lib/copter.js

```diff
@@ -24,9 +23,19 @@ class Copter {
   this.downpull = - 6;
   }
 
-  crash() {
-    $('.game-canvas').hide();
-    $('.end-game').show();
+  checkForCollision(collider) {
+    if (this.x < collider.x + collider.width &&
```

Comment:
I would pull the contents of the if statement into another helper function. Would make it easier to test.